docs/docs-content/clusters/edge/site-deployment/deploy-custom-registries/deploy-external-registry.md

```diff
@@ -0,0 +1,120 @@
+---
+sidebar_label: "Deploy Cluster with a Private External Registry"
+title: "Deploy Cluster with a Private External Registry"
+description: "Instructions for deploying an Edge cluster with a private external registry."
+hide_table_of_contents: false
+sidebar_position: 60
+tags: ["edge"]
+---
+
+Palette Edge provides support for downloading images from authenticated external registries. You can instruct the
+Palette agent to download images from an authenticated external registry by specifying the address and the credentials
+for the registry in the user data used to build your Edge Installer ISO.
+
+Once you specify an external registry, images for all elements of the cluster are expected to be in the external
+registry. This includes the provider images, images for the network and storage layer, and images for all application
+layers. All images specified in the cluster profile will have their registry URL replaced by the registry URL of the
+external image registry.
+
+:::tip
+
+You can use a private external registry together with a local Harbor image registry. All images from the external
+private registry will be stored in the local Harbor registry after the initial download, which allows you to reduce
+bandwidth use and protect against outages. For more information, refer to
```

Comment:
```suggestion
bandwidth use and gain protection against outages. For more information, refer to
```

Reply:
Or maybe protect your system or something? Protect feels a bit out of place here